controller/api/public/http_server.go

```diff
@@ -312,6 +312,7 @@ func NewServer(
 	discoveryClient discoveryPb.DiscoveryClient,
 	k8sAPI *k8s.API,
 	controllerNamespace string,
+	clusterDomain string,
```

Comment:
This shouldn't be necessary. See comment on reading the cluster domain from the `linkerd-config` config map.

Reply:
Removed.